models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1591 @@
+If you experience these issues, a helpful troubleshooting guide is 
+located `here. <https://docs.dart.ucar.edu/en/latest/guide/dart-quality-control.html>`__   
+
+
+Matlab Diagnostics
+------------------
+
+Once you have confirmed that the assimilation has been completed
+reasonably well as outlined by the steps above, the DART package includes
+a wide variety of Matlab diagnostic scripts that provide a more formal evaluation
+of assimilation performance.  These diagnostics can provide clues
+to further maximize performance through adjustments of the DART settings 
+(localization, inflation, etc.). The full suite of diagnostic scripts can be found
+at this path in your DART installation (``<dartroot>/diagnostics/matlab``) with supporting 
+documentation found `here. <https://docs.dart.ucar.edu/en/latest/guide/matlab-observation-space.html>`__
+
+
+.. Note::
+   
+ Additional scripts that are designed for CLM output visualization
+ can be found here (``<dartroot>/models/clm/matlab``).  The ``clm_get_var.m`` and ``clm_plot_var.m``
+ scripts are designed to re-constitute a vector-based file (e.g. restart.nc) into 
+ gridded averages to allow viewing of spatial maps.  These scripts are helpful to
+ view the model update by DART (innovations). An example of how to implement these
+ scripts can be found here (``<dartroot>/models/clm/matlab/README.txt``).
+
+          
+Here we provide instructions to execute two highly recommended matlab scripts.
+First, the ``plot_rmse_xxx_evolution.m`` script provides a time series of 
+assimilation statistics of 1) observation acceptance, 2) RMSE between
+the observations and the expected observation (derived from the CLM state),
+and 3) a third statistic of your choosing (we recommended 'total spread').
+
+The observation acceptance statistic compares the number of observations assimilated
+versus the number of observations available for your domain.  In general, it
+is desirable to assimilate the majority of observations that are available.
+The RMSE statistic quantifies the mismatch between the observations and the
+CLM state.  A successful assimilation reduces the RMSE, thus reducing the mismatch
+between the observations and the CLM state. Finally the total spread provides
+contributions from the ensemble spread and observation error variance. This value
+should be comparable to the RMSE.
+
+To execute ``plot_rmse_xxx_evolution.m`` do the following:
+
+::
+ 
+ > cd <dartroot>/models/clm/work
+
+Confirm the DART executables used for the matlab diagnostics exist.
+These should have been compiled during Step 11 of this tutorial.
+The important DART executables for the diagnostics are
+``obs_diag`` and ``obs_seq_to_netcdf``.  If they do not exist,
+perform the ``./quickbuild.csh -mpi`` command to create them.
+
+Next generate a text file that includes all the ``obs_seq*.final``
+files from the tutorial simulation
+
+::
+ 
+ > ls <rundir>/*final > obs_seq_files_tutorial.txt 
```

Comment:
rundir here is a ways away from where it was defined.  It might be worth a reminder of where rundir is. 

```
setenv rundir           ${cime_output_root}/run
```

```suggestion
 ls <rundir>/*final > obs_seq_files_tutorial.txt 
```